js/main.js

```diff
@@ -2,14 +2,72 @@ var rightAnswer = false;
 var currentID = 0;
 var pokemonName = null;
 var randomIDList = shuffle(1, 10);
+var intervalIDUserTimer = null;
+var intervalIDFiveSecondTimer = null;
+var time = null;
+var seconds = 59;
+var imgSeconds = 5;
 var $pokemonImg = document.querySelector('.pokemon-img');
-var $answerBox = document.querySelector('.answer-input');
+var $answerBox = document.querySelector('.answer');
 var $skipButton = document.querySelector('.button-skip');
+var $timeChoice = document.querySelector('.time-form');
+var $homeContainer = document.querySelector('.home');
+var $quizContainer = document.querySelector('.quiz');
+var $timer = document.querySelector('.timer');
+var $fiveSecondTimer = document.querySelector('.five-second-timer');
 
-addEventListener('load', getPokemon);
 $answerBox.addEventListener('keydown', getNextPokemon);
 $answerBox.addEventListener('input', correctPokemon);
 $skipButton.addEventListener('click', skipPokemon);
+$timeChoice.addEventListener('submit', getTime);
+
+
+function getTime(event) {
+  event.preventDefault();
+  time = parseInt($timeChoice.elements.time.value);
+  if (Object.is(time, NaN)) {
+    $timeChoice.elements.time.value = null;
+    $timeChoice.elements.time.placeholder = 'ENTER A NUMBER';
+  } else {
+    $homeContainer.className = 'container home hidden';
+    $quizContainer.className = 'container quiz';
+
+    intervalIDFiveSecondTimer = setInterval(countDown5Second, 1000);
+  }
+}
+
+function countDown5Second() {
+  imgSeconds--;
```

Comment:
Does `imgSeconds` ever get set back to 5? What if the user takes the quiz again? Does it start at `0`?